packages/react/src/components/Dropzone/Files.tsx

```diff
@@ -0,0 +1,72 @@
+import { useEffect } from "react";
+
+import { WorkspaceElement } from "edifice-ts-client";
+
+import { useDropzoneContext } from "./Dropzone";
+import useHandleFile from "../../hooks/useHandleFiles/useHandleFiles";
+import { UploadCard } from "../../multimedia";
+import { custumSize } from "../../utils/fileSize";
+
+export interface FilesProps {
+  uploadFile: WorkspaceElement | File;
+  index: number;
+  handleDelete: any;
+}
+
+const Files = ({ uploadFile, index, handleDelete }: FilesProps) => {
+  const { statusUpload, handleSave } = useHandleFile();
+  const { setUploadFiles } = useDropzoneContext();
+
+  const callHandleSave = async () => {
```

Comment:
@Headan A harmoniser d'après la [convention](https://edifice-community.atlassian.net/wiki/spaces/FK/pages/3451879476/Convention#Naming-Events-Handlers) en place
+ un autre [lien](https://javascript.plainenglish.io/handy-naming-conventions-for-event-handler-functions-props-in-react-fc1cbb791364)